tests/installation/bootloader_svirt.pm

```diff
@@ -8,14 +8,20 @@
 # without any warranty.
 
 use base "installbasetest";
-
 use strict;
 use warnings;
-
-use File::Basename;
-
 use testapi;
 use utils;
+use File::Basename;
+
+sub is_netinstall() {
+    if (!is_jeos && !get_var('ISO')) {
+        return 1;
+    }
+    else {
+        return 0;
+    }
+}
 
```

Comment:
Network installations use "if (get_var("NETBOOT"))..."
ISO variable is defined there, as they boot from minimal ISO.